Doc/conf.py

```diff
@@ -97,6 +97,7 @@
     ('c:func', 'vsnprintf'),
     # Standard C types
     ('c:type', 'FILE'),
+    ('c:type', 'int'),
```

Comment:
This shouldn't be needed (see other comments)
```suggestion
```

Reply:
Yeah, that's what I thought. It was a last resort to try and fix the CI. Thanks.